changelog/db.changelog-1.25.xml

```diff
@@ -0,0 +1,221 @@
+      <column name="schemeId" value="4" />
+      <column name="name" value="Top up" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="55" />
+      <column name="schemeId" value="4" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="56" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="DAX rejection" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="57" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Awaiting debt enrichment" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="58" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Ex-gratia" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="59" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Hardship case" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="60" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Non-payable" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="61" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Withdrawal" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="62" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Manual payment" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="63" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Bridging payments" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="64" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Other admin" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="65" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Bank account anomaly" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="66" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Recovery" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="67" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Top up" />
+    </insert>
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="68" />
+      <column name="schemeId" value="5" />
+      <column name="name" value="Manual ledger hold" />
+    </insert>
+  </changeSet>
```

Comment:
All have:

- DAX rejection
- Awaiting debt enrichment
- Ex-gratia
- Hardship case 
- Non-payable
- Withdrawal 
- Manual payment 
- Bridging payments
- Other admin
- Bank account anomaly
- Recovery 
- Top up 
- Manual ledger hold

But none have:
- Migrated hold

Introduced in 1.14